velox/dwio/common/ColumnVisitors.h

```diff
@@ -137,6 +137,98 @@ class ExtractToGenericHook {
   ValueHook* hook_;
 };
 
+class ExtractStringDictionaryToGenericHook {
+ public:
+  using HookType = ValueHook;
+  static constexpr bool kSkipNulls = false;
+
+  ExtractStringDictionaryToGenericHook(
+      ValueHook* hook,
+      RowSet rows,
+      RawScanState* state)
+      : hook_(hook), rows_(rows), state_(state) {}
+
+  bool acceptsNulls() const {
+    return hook_->acceptsNulls();
+  }
+
+  template <typename T>
+  void addNull(vector_size_t rowIndex) {
+    hook_->addNull(rowIndex);
+  }
+
+  template <typename V>
+  void addValue(vector_size_t rowIndex, V value) {
```

Comment:
Why would it take anything else than int32 at compile time?

Reply:
Remove this code, and the following exception will occur during compilation
```
/data/home/wypb/data/code/apache/velox/./velox/dwio/common/ColumnVisitors.h:222:69: error: cannot convert ‘folly::Range<const char*>’ to ‘int32_t’ {aka ‘int’}
  222 |         ? extractStringDictionaryToGenericHook_->addValue(rowIndex, value)
      |                                                                     ^~~~~
      |                                                                     |
      |                                                                     folly::Range<const char*>
```